mergify_engine/config.py

```diff
@@ -197,9 +197,25 @@ def ApplicationAPIKeys(v: str) -> typing.Dict[str, ApplicationAPIKey]:
         voluptuous.Required(
             "REDIS_QUEUE_WEB_MAX_CONNECTIONS", default=None
         ): voluptuous.Any(None, voluptuous.Coerce(int)),
-        voluptuous.Required("STORAGE_URL", default="redis://localhost:6379?db=8"): str,
+        # NOTE(sileht): Unused anymore, but keep to detect legacy onpremise installation
+        voluptuous.Required("STORAGE_URL", default=None): voluptuous.Any(None, str),
+        voluptuous.Required(
+            "DEFAULT_REDIS_URL", default="redis://localhost:6379?db=8"
```

Comment:
```suggestion
            "DEFAULT_REDIS_URL", default="redis://localhost:6379"
```
or just don't set any default; the current one is going to fail.